WordPress/gutenberg
# Removed line break within the code block

## What?

There is a line break within the code block. 
## Why?

I have removed the line breaks as the line break during the class are not necessary.
## How?


## Testing Instructions





Navigate to: https://developer.wordpress.org/block-editor/explanations/architecture/styles/#block-styles 

See the unnecessary line breaks within the second code block.

### Testing Instructions for Keyboard


## Screenshots or screencast
